hikari/guilds.py

```diff
@@ -414,6 +435,9 @@ class Member(users.User):
         This takes precedence over [`hikari.guilds.Member.avatar_hash`][].
     """
 
+    guild_flags: typing.Union[GuildMemberFlags, int] = attrs.field(eq=False, hash=False, repr=False)
```

Comment:
But there are also user flags and since member is a subclass of user the property flags returns the user flags and i figured it would be better to name this guild flags because then it would break code where people used Member.flags to get the user flags